wordpress-mobile/AztecEditor-iOS
# Fix ordered list big numbers display

Refs #https://github.com/wordpress-mobile/gutenberg-mobile/issues/1679

This PR increase the spacing provide for items marker to ensure that items up to four digits are displaying using the default font size. It also makes sure that numbers that are large don't overlap the text content of the list.

![Simulator Screen Shot - iPhone 11 Pro - 2020-01-17 at 12 15 03](https://user-images.githubusercontent.com/651601/72611832-03e1f500-3923-11ea-9b54-3678454d20e3.png)


To test:
 - Run the demo app
 - Select the big lists demo
 - Check that items are displayed and when they large they don't overlap the list text but instead start to get cropped on the left side.
